src/pyscipopt/scip.pxi

```diff
@@ -4120,6 +4130,39 @@ cdef class Model:
         PY_SCIP_CALL(SCIPdelVar(self._scip, var.scip_var, &deleted))
         return deleted
 
+    def aggregateVars(self, Variable varx, Variable vary, scalarx=1.0, scalary=-1.0, rhs=0.0):
+        """
+        Aggregate varx and vary: calls SCIPaggregateVars and returns
```

Comment:
Could you please add more info on what SCIPaggregateVars does? a formula would be very helpful here so that one wouldn't need to go the C-API documentation.